pkg/storage/metrics.go

```diff
@@ -28,288 +28,424 @@ import (
+	metaMaxCommandQueueTreeSize = metric.Metadata{
+		Name: "replicas.commandqueue.maxtreesize",
 		Help: "Largest number of intervals in any CommandQueue's interval tree"}
-	metaMaxCommandQueueOverlaps = metric.Metadata{Name: "replicas.commandqueue.maxoverlaps",
+	metaMaxCommandQueueOverlaps = metric.Metadata{
+		Name: "replicas.commandqueue.maxoverlaps",
 		Help: "Largest number of overlapping commands seen when adding to any CommandQueue"}
-	metaCombinedCommandQueueSize = metric.Metadata{Name: "replicas.commandqueue.combinedqueuesize",
+	metaCombinedCommandQueueSize = metric.Metadata{
+		Name: "replicas.commandqueue.combinedqueuesize",
 		Help: "Number of commands in all CommandQueues combined"}
-	metaCombinedCommandWriteCount = metric.Metadata{Name: "replicas.commandqueue.combinedwritecount",
+	metaCombinedCommandWriteCount = metric.Metadata{
+		Name: "replicas.commandqueue.combinedwritecount",
 		Help: "Number of read-write commands in all CommandQueues combined"}
-	metaCombinedCommandReadCount = metric.Metadata{Name: "replicas.commandqueue.combinedreadcount",
+	metaCombinedCommandReadCount = metric.Metadata{
+		Name: "replicas.commandqueue.combinedreadcount",
 		Help: "Number of read-only commands in all CommandQueues combined"}
 
 	// Range metrics.
-	metaRangeCount = metric.Metadata{Name: "ranges",
+	metaRangeCount = metric.Metadata{
+		Name: "ranges",
 		Help: "Number of ranges"}
-	metaUnavailableRangeCount = metric.Metadata{Name: "ranges.unavailable",
+	metaUnavailableRangeCount = metric.Metadata{
+		Name: "ranges.unavailable",
 		Help: "Number of ranges with fewer live replicas than needed for quorum"}
-	metaUnderReplicatedRangeCount = metric.Metadata{Name: "ranges.underreplicated",
+	metaUnderReplicatedRangeCount = metric.Metadata{
+		Name: "ranges.underreplicated",
 		Help: "Number of ranges with fewer live replicas than the replication target"}
 
 	// Lease request metrics.
-	metaLeaseRequestSuccessCount  = metric.Metadata{Name: "leases.success"}
-	metaLeaseRequestErrorCount    = metric.Metadata{Name: "leases.error"}
-	metaLeaseTransferSuccessCount = metric.Metadata{Name: "leases.transfers.success"}
-	metaLeaseTransferErrorCount   = metric.Metadata{Name: "leases.transfers.error"}
-	metaLeaseExpirationCount      = metric.Metadata{Name: "leases.expiration"}
-	metaLeaseEpochCount           = metric.Metadata{Name: "leases.epoch"}
+	metaLeaseRequestSuccessCount = metric.Metadata{
+		Name: "leases.success",
+		Help: "Number of successful lease requests"}
+	metaLeaseRequestErrorCount = metric.Metadata{
+		Name: "leases.error",
+		Help: "Number of failed lease requests"}
+	metaLeaseTransferSuccessCount = metric.Metadata{
+		Name: "leases.transfers.success",
+		Help: "Number of successful lease transfers"}
+	metaLeaseTransferErrorCount = metric.Metadata{
+		Name: "leases.transfers.error",
+		Help: "Number of failed lease transfers"}
+	metaLeaseExpirationCount = metric.Metadata{
+		Name: "leases.expiration",
+		Help: "TODO(marc): fill me"}
```

Comment:
I've found it mildly useful at times to see that the number of expiration-based leases in a cluster dropped down from its steady state, indicating a system range is unavailable. It's not useful for much else, though, and other metrics could also cover that case.